django_redis/client/default.py

```diff
@@ -202,15 +221,28 @@ def incr_version(self, key, delta=1, version=None, client=None):
         self.delete(old_key, client=client)
         return version + delta
 
-    def add(self, key, value, timeout=DEFAULT_TIMEOUT, version=None, client=None):
+    def add(
+        self,
+        key: Any,
+        value: Any,
+        timeout: Any = DEFAULT_TIMEOUT,
+        version: Optional[Any] = None,
```

Comment:
Should these not be constrained more?

Reply:
Well, yes and no. Technically key and value could be `ByteStr`, I was not sure because you could decide to use many types and they will be valid for redis. What do you suggest? I would keep it this way and discuss about it elsewhere, at the time I even asked on django stubs repository because it was not typed even there, don't know now, I will check

Reply:
I see, well I'm not sure why things are loosely defined on django-stubs, and I'm fine with leaving `Any`, but not sure if that's much better than leaving it undefined :confused: